pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -85,7 +90,7 @@ public enum BrokerMeter implements AbstractMetrics.Meter {
   REQUEST_DROPPED_DUE_TO_ACCESS_ERROR("requestsDropped", false),
 
   GROUP_BY_SIZE("queries", false),
-  TOTAL_SERVER_RESPONSE_SIZE("queries", false),
+  TOTAL_SERVER_RESPONSE_SIZE("bytes", false),
```

Comment:
Will this potentially cause backward compatible issue?

Reply:
I can rollback this to be sure.